python/lsst/ip/isr/measureCrosstalk.py

```diff
@@ -195,7 +195,7 @@ def parseAndRun(cls, *args, **kwargs):
             pickle.dump(resultList, open(results.parsedCmd.dumpRatios, "w"))
         return task.reduce(resultList)
 
-    def run(self, dataRef):
+    def runDataRef(self, dataRef):
```

Comment:
I think this is a case where `runDataRef` should call `run` since `run` can do useful work on the unpersisted exposure.

Reply:
The work of converting tasks to have the work done in run, and the runDataRef be the entry point for command line task is planned for after this ticket is complete. We chose to do the big outward facing api change first, and save any refactoring work for another ticket.

Reply:
This would be a trivial change. Most of the other renames do not lend themselves to adding `run` methods (at least that I saw in the packages I looked at) but this one did.

Reply:
In the interest in merging the changes for this ticket that refactoring depends on, @yalsayyad  has helped by opening a ticket on this refactorization.  DM-15311 covers this change, and I'll begin work on it next week.